packages/trace-viewer/src/ui/sourceTab.css

```diff
@@ -31,3 +31,13 @@
   box-shadow: var(--vscode-scrollbar-shadow) 0 6px 6px -6px;
   z-index: 10;
 }
+
+.copy-icon.codicon {
```

Comment:
This will affect the entire app, not just this local icon. Did you mean to scope it?

Reply:
oops, my bad. I have fixed this now